bdsx/bds/implements.ts

```diff
@@ -1120,6 +1121,21 @@ Actor.prototype.setOwner = procHacker.js("?setOwner@Actor@@UEAAXUActorUniqueID@@
 Actor.prototype.getVariant = procHacker.js("?getVariant@Actor@@QEBAHXZ", int32_t, { this: Actor });
 Actor.prototype.setVariant = procHacker.js("?setVariant@Actor@@QEAAXH@Z", void_t, { this: Actor }, int32_t);
 Actor.prototype.setTarget = procHacker.jsv("??_7Actor@@6B@", "?setTarget@Actor@@UEAAXPEAV1@@Z", void_t, { this: Actor }, Actor);
+Actor.prototype.playAnimation = function (animation, options = {}) {
+    const pk = AnimateEntityPacket.allocate();
+    pk.mAnimation = animation;
+    pk.mNextState = options.nextState ?? "default";
+    pk.mBlendOutTime = options.blendOutTime ?? 0;
+    pk.mStopExpression = options.stopExpression ?? "query.any_animation_finished";
+    pk.mStopExpressionVersion = 1;
+    pk.mController = options.controller ?? "__runtime_controller";
```

Comment:
is there any reference about this? : `__runtime_controller`

Reply:
All default values refer to packets sent when using `/playanimation` command without options.

Reply:
![image](https://github.com/bdsx/bdsx/assets/41989402/e17d1d20-6c99-468b-8720-f1d5ad0377cd)

Reply:
oh I missed the picture, thanks.